decidim-admin/app/forms/decidim/admin/organization_form.rb

```diff
@@ -9,11 +9,23 @@ class OrganizationForm < Rectify::Form
 
       mimic :organization
 
+      attribute :current_organization, Decidim::Organization
+      attribute :current_user, Decidim::User
+
       attribute :name, String
+      attribute :default_locale, String
       translatable_attribute :description, String
 
       validates :name, presence: true
-      translatable_validates :description, presence: true
+      validates :default_locale, presence: true
+      validates :default_locale, inclusion: { in: :available_locales }
+      validates :description, translatable_presence: true
+
+      private
+
+      def available_locales
```

Comment:
I didn't think about delegating, sincerely, but there's no need to make this public so I think it's OK as it is.